### PR TITLE
[Fix]: Read aloud will stop on close tab

### LINKED ISSUE #3740: [Bug] Read aloud won't stop for closed tabs
**Describe the bug**

If the read-aloud feature is running and reading through the page, it will not stop when the specific tab or all tabs is closed.

**Expected behavior**

It should STOP when the tab is closed which it was reading

**Steps to reproduce the behavior:**
1. Open a ZIM and start Read aloud from top right corner.

**Screenshots**

**Environment**
- Version of Kiwix Android : 3.9.1
- Device : REALME GT
- OS version : 13

**Logs**

### PR DESCRIPTION
**Pull request by Sagar0-0**
Fixes #3740 

Used onReadAloudStop method on close tab

**Screenshots**

### CODE REVIEW

**Review by MohitMaliFtechiz — CHANGES_REQUESTED** [review]
@Sagar0-0 Please make changes in this PR according to these two comments https://github.com/kiwix/kiwix-android/issues/3740#issuecomment-1991308858 https://github.com/kiwix/kiwix-android/issues/3740#issuecomment-1991390665

**Comment by Sagar0-0** [comment]
@MohitMaliFtechiz please confirm the final changes.

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt [hunk 1]
```diff
@@ -1192,6 +1194,7 @@ abstract class CoreReaderFragment :
   }
 
   private fun closeTab(index: Int) {
+    if (currentTtsWebViewIndex == index) onReadAloudStop()
```
Comment (MohitMaliFtechiz) [inline core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt:1197]: Please use `if` condition with curly braces to maintain the project's code style.

**Review by MohitMaliFtechiz — CHANGES_REQUESTED** [review]
@Sagar0-0 Just a small change, and then we are good to merge.

File: core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt (hunk 1 above)
Comment (Sagar0-0) [reply core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt]: I have made the changes @MohitMaliFtechiz

**Review by MohitMaliFtechiz — APPROVED** [review]
@Sagar0-0 Thanks :tada:

**Pull request merged**